image/postgres/Dockerfile

```diff
@@ -1,9 +1,10 @@
+ARG POSTGRES_REGISTRY=quay.io
+ARG POSTGRES_BASE_IMAGE=centos/centos:stream9
 ARG BASE_REGISTRY=registry.access.redhat.com
-ARG BASE_IMAGE=ubi8-minimal
-ARG POSTGRES_BASE_IMAGE=ubi8
-ARG BASE_TAG=8.7
+ARG BASE_IMAGE=ubi9-minimal
+ARG BASE_TAG=9.2
```

Comment:
This starts to deviate from downstream. It would be great to override the docker build step so that we can pass these args and only z would use centos/rhel9

Reply:
added commit to revert back to ubi8 on all platforms except s390x.  Added necessary docker build args for s390x to Makefile.